addons/mail/static/src/views/web/fields/html_composer_message_field/content_expandable_plugin.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="mail.ExpandableButton">
+        <br t-if="newLine"/>
+        <div contenteditable="false">
+            <button id="viewMore" class="badge rounded-pill border-0 m-1 py-0 px-1">
```

Comment:
Could you instead of using id have a class so that you can identify the button more easily? Like `o-mail-Message-viewMore-btn` or something else if you have a better idea.